pages/article/[id].js

```diff
@@ -342,12 +349,20 @@ function ArticlePage() {
               </Infos>
             </header>
             <CardContent>
-              {nl2br(
-                linkify(text, {
-                  props: {
-                    target: '_blank',
-                  },
-                })
+              {text ? (
+                nl2br(
+                  linkify(text, {
+                    props: {
+                      target: '_blank',
+                    },
+                  })
+                )
+              ) : (
+                <img
+                  className={classes.bannerImage}
+                  src={attachmentUrl}
+                  alt="image"
+                ></img>
```

Comment:
Same as above.

I think `bannerImage` is a bit too wide (100% width) on desktop. I think we can set `max-width` & `max-height` on them, and use `<a>` to link to the original image instead.
```suggestion
              {attachmentUrl && (
                <img
                  className={classes.bannerImage}
                  src={attachmentUrl}
                  alt="image"
                />
              }
              {text && (
                nl2br(
                  linkify(text, {
                    props: {
                      target: '_blank',
                    },
                  })
                )
              )}
```